sdk/include/xdk/setypes.h

```diff
@@ -146,6 +146,14 @@ typedef struct _SE_IMPERSONATION_STATE {
 #define SEF_AVOID_OWNER_RESTRICTION         0x1000
 #define SEF_MACL_VALID_FLAGS (SEF_MACL_NO_WRITE_UP | SEF_MACL_NO_READ_UP | SEF_MACL_NO_EXECUTE_UP)
 
+/* Privilege token filtering flags */
+#define DISABLE_MAX_PRIVILEGE 0x1
+#define SANDBOX_INERT         0x2
+#if (NTDDI_VERSION >= NTDDI_LONGHORN)
+#define LUA_TOKEN             0x4
+#define WRITE_RESTRICTED      0x8
+#endif
+
 $endif (_WDMDDK_ || _WINNT_)
```

Comment:
Where you currently have these definitions, they will end up in both `wdm.h` and `winnt.h`. They should move to a section that only has `$if (_WINNT_)`